src/Engine/ProtoScript/Runners/LiveRunner.cs

```diff
@@ -1680,7 +1680,7 @@ private void CompileAndExecuteForDeltaExecution(List<AssociativeNode> astList)
             CompileAndExecute(dispatchASTList);
             PostExecution();
             timer.Stop();
-            ExecutionTime = timer.Elapsed;
+            CompileAndExecutionTime = (new TimeSpan(0), timer.Elapsed);
```

Comment:
To generate compile time only and execution time only for old engine, we can add another timer in `bool LiveRunner.CompileAndExecute(string code)`, something like:
```
private bool CompileAndExecute(string code)
        {
            // TODO Jun: Revisit all the Compile functions and remove the blockId out argument
// compiler timer start
            bool succeeded = runner.CompileAndGenerateExe(code, runnerCore, new ProtoCore.CompileTime.Context());
// compile timer end
            if (succeeded)
            {
// execution timer start
                Execute(!string.IsNullOrEmpty(code));
// execution timer end
            }
            return succeeded;
        }
```